src/views/LoginView.vue

```diff
@@ -38,34 +44,35 @@ export default {
     return {
       username : "",
       password : "",
+      role : "",
+      // login : true
+    }
+  },
+  computed : {
+    login(){
+      return this.$route.name === 'login' ? true : false
```

Comment:
no need to use a conditional operator, only write route === 'login' it will automatically return true or false

Reply:
fixed it